models/mart/core/fct_orders.sql

```diff
@@ -0,0 +1,11 @@
+
+with orders as (
+    SELECT * FROM {{ref('stg_jaffle_shop__orders')}}
+)
+
```

Comment:
My personal / Fishtown SQL style would say to use another CTE here
```sql
payments as (

    select * from {{ref('stg_stripe_payments')}}

)
```
so that you can reference this by its alias `payments` down below. We like to think of these as analogous to a python `import` that goes at the top of every file, telling you which modules/assets it's pulling in as dependencies.